### DISCUSSION TITLE
Intel param

### DISCUSSION THREAD

**Discussion by NoAce4U** [Vdauphin/HeartsAndMinds#1165] [Question and answer]
`class btc_p_info_chance { // Intel from dead bodies chance:
        title = __EVAL(format ["      %1", localize "STR_BTC_HAM_PARAM_CACHE_DEADINFO"]);
        values[]={0,10,20,30,40,50,60,70,80,90,100};
        texts[]={"100 %","90 %","80 %","70 %","60 %","50 %","40 %","30 %","20 %","10 %","0 %"};`
        
        Is this supposed to be backward?

**Comment by Vdauphin** [comment]
Hello,
https://github.com/Vdauphin/HeartsAndMinds/blob/cfb6ccd26b8d358135d82dedf96ed80d475316fb/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/mil/unit_killed.sqf#L35
Cheers

**Reply by NoAce4U** [reply]
I guess I don't quite understand

`values[]={0,10,20,30,40,50,60,70,80,90,100};`

But

`texts[]={"100 %","90 %","80 %","70 %","60 %","50 %","40 %","30 %","20 %","10 %","0 %"};`

Is 0 = 100%?

**Reply by NoAce4U** [reply]
OH, I get it now. Ignore my stupidity
